### PR TITLE
SLM-14 redirects

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

File: server/middleware/link/requestLinkAuthorised.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { RequestHandler } from 'express'
+
+export default function requestLinkAuthorised(): RequestHandler {
```
Comment (mikehalmamoj) [inline server/middleware/link/requestLinkAuthorised.ts:3]: I need a unit test for this - coming in next PR

File: assets/js/mojFrontendInit.js [hunk 2]
```diff
@@ -0,0 +1 @@
+window.MOJFrontend.initAll()
```
Comment (smcveigh941) [inline assets/js/mojFrontendInit.js:1]: Have you put this in a `script` tag in your html layout?

**Review by smcveigh941 — APPROVED** [review]
LGTM

File: assets/js/mojFrontendInit.js (hunk 2 above)
Comment (mikehalmamoj) [reply assets/js/mojFrontendInit.js:1]: Yup.

**Review by nathanrussell-moj-digital — APPROVED** [review]

**Pull request merged**